contrib/scripts/protocgen.sh

```diff
@@ -13,7 +13,7 @@ protoc_gen_gocosmos() {
   # get protoc executions
   go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos@latest 2>/dev/null
 
-  # get cosmos sdk from github
+  # get cosmos sdk
```

Comment:
This was a trivial change added to make the workflow run

Reply:
XD